### ISSUE TITLE
Use arguments instead of configuration file

### ISSUE THREAD

**Issue by yukiisbored** [Abhi2424shek/img_checker#25]
I think it's a good idea to have a way to use arguments instead of using the configuration file for something like [coala](github.com/coala/coala), git hooks, etc. You can achieve this easily with `argparse`.

**Comment by Abhi2424shek** [comment]
If there are more directories the number of arguments would be long and that's not good.

**Comment by yukiisbored** [comment]
Yes, but that would make something like git hooks and coala overly complicated. The reason for this is for usability with other programs/scripts not humans

**Comment by Abhi2424shek** [comment]
Can we have another tweak to define the path of the config file?

**Comment by yukiisbored** [comment]
No, using a config file make things complicated.

**Comment by Abhi2424shek** [comment]
We can't put it inside coala besides when coala uses it users would have trouble.

**Comment by yukiisbored** [comment]
Why? You can still have the configuration file, in fact do that on default when there's no arguments defined. You can add something like `--ignore-config` to Ignore the config or when "config replacing" arguments is defined ignore the config or add it.

**Comment by yukiisbored** [comment]
It's not only useful for `coala`. It's useful for users that just want to do one-time checks without writing/having the config leaving a trail.

**Comment by Abhi2424shek** [comment]
Ok we will implement it!

**Comment by abishekvashok** [comment]
optparse
